ray-on-gke/tpu/kuberay-tpu-webhook/main.go

```diff
@@ -262,20 +342,6 @@ func validateRayCluster(admissionReview *admissionv1.AdmissionReview) (*admissio
 	workerGroupSpecs := raycluster.Spec.WorkerGroupSpecs
```

Comment:
Line 333 calls `klog.Fatalf` and kills the program, can we update it to return an error instead?

Reply:
Done in [25bcdb8](https://github.com/GoogleCloudPlatform/ai-on-gke/pull/723/commits/25bcdb8a906abf05c879c6888eb5e8dcee058ad2)